react/week3/todolist/src/Components/Timer.js

```diff
@@ -0,0 +1,21 @@
+import React, {useState, useEffect} from 'react'
+
+const Timer = () => {
+    const [timer, setTimer] = useState(0)
+
+    useEffect(() => {
+        const startTimer = setTimeout(() => {
+            setTimer((timer) => timer + 1)
+        }, 1000)
+        return () => clearTimeout(startTimer)
```

Comment:
Use setInterval as setTimeout just means delay by X time.

The reason it may be working here is because you are "re-rendering" the component and [listeningToState] which makes it do it every 1000ms in a loop.